engine/src/quicktime.cpp

```diff
@@ -219,8 +219,17 @@ static bool path_to_dataref(MCStringRef p_path, DataReferenceRecord& r_rec)
 {
 	bool t_success = true;
 	CFStringRef t_cf_path = NULL;
-	t_cf_path = CFStringCreateWithCString(NULL, MCStringGetCString(p_path), kCFStringEncodingWindowsLatin1);
-	t_success = (t_cf_path != NULL);
+
+	// SN-2015-05-14: [[ MCStringGetCString Removal ]] Use a UTF-8 string
+	MCAutoStringRefAsUTF8String t_cstring_path;	
```

Comment:
That suggests the quicktime on Windows exports the CFStringCreate* APIs though - otherwise the code wouldn't link ;)

It's probably worth (in this instance) tweaking the foundation-auto header so that the Auto class is defined on Windows - it cleans up this code slightly.